tensorflow/core/platform/default/logging.h

```diff
@@ -71,11 +76,18 @@ class LogMessageFatal : public LogMessage {
 
 #define LOG(severity) _TF_LOG_##severity
 
-// TODO(jeff): Define a proper implementation of VLOG_IS_ON
+#ifdef IS_MOBILE_PLATFORM
+// Turn VLOG off when under mobile devices for considerations of binary size.
 #define VLOG_IS_ON(lvl) ((lvl) <= 0)
+#else
+// Otherwise, Set TF_CPP_MIN_VLOG_LEVEL environment to update minimum log level
+// of VLOG
+#define VLOG_IS_ON(lvl) \
+  ((lvl) <= ::tensorflow::internal::LogMessage::MinVLogLevel())
+#endif
 
 #define VLOG(lvl)      \
-  if (VLOG_IS_ON(lvl)) \
+  if (TF_PREDICT_FALSE(VLOG_IS_ON(lvl))) \
```

Comment:
@leftnoteasy I am sorry to comment in this `merged` PR, i'm just wondering why predict `FALSE` here?